test/unit/node_stream_spec.js

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* globals __non_webpack_require__ */
+
+import { isNodeJS } from '../../src/shared/util';
+import { PDFNodeStream } from '../../src/display/node_stream';
+
+describe('node_stream', function() {
+  let pdf1 =
+    'http://mozilla.github.io/pdf.js/web/compressed.tracemonkey-pldi-09.pdf';
+  let pdfLength = 1016315;
+  let pdf2;
+  if (!isNodeJS()) {
+    pdf2 = new URL('../pdfs/tracemonkey.pdf', window.location).href;
+  } else {
+    let path = __non_webpack_require__('path');
+    let url = __non_webpack_require__('url');
+    pdf2 =
+      url.parse(path.join(process.cwd(), './test/pdfs/tracemonkey.pdf')).href;
```

Comment:
> I was expecting a file:// prefix, because otherwise it would not be a URL.

Then I think adding `file://` with `process.cwd()` will help, something like : `url.parse(path.join('file://' + process.cwd(), './test/pdfs/tracemonkey.pdf'))`